### PR TITLE
docs: improve permission page

### PR DESCRIPTION
**Pull request by RomainGoussault**

### CODE REVIEW

File: docs/source/documentation/concepts.rst [hunk 1]
```diff
@@ -130,7 +130,7 @@ In the following tables, the asset is registered by orgA with the permissions:
 Parallelization
 ---------------
 
-There are two ways to run several tasks in parallel on a same organization. The first one, named vertical scaling, is when several tasks are run in parallel on the same machine. The second one, horizontal scaling, is when several tasks are run in parallel on several machines belonging to the same organization.
+There are two ways to run several tasks in parallel on a same organization with Substra. The first one, named vertical scaling, is when several tasks are run in parallel on the same machine. The second one, horizontal scaling, is when several tasks are run in parallel on several machines belonging to the same organization.
```
Comment (RomainGoussault) [inline docs/source/documentation/concepts.rst:133]: This addition is to make clear those two ways are implemented today.

File: docs/source/documentation/concepts.rst [hunk 2]
```diff
@@ -76,9 +76,9 @@ Permissions for a user to download an asset
 
 Users of a organization can export (aka download) the following elements from Substra to their local environment:
 
-* the opener of a dataset if the organization has process permissions on the dataset
-* the archive of a function if the organization has process permissions on the function
-* the model outputted by a task if the organization has process permissions on the model and if this type of export has been enabled at deployment for the organization (environment variable model_export_enabled should be set to True)
+* The opener of a dataset if the organization has process permissions on the dataset.
+* The archive of a function if the organization has process permissions on the function.
+* The model outputted by a task if the organization has process permissions on the model **AND** if this type of export has been enabled at deployment for the organization. This means the environment variable ``model_export_enabled`` should be set to ``True`` in the organization's backend.
```
Comment (SdgJlbl) [inline docs/source/documentation/concepts.rst:81]:
```suggestion
* The model outputted by a task if the organization has process permissions on the model **AND** if this type of export has been enabled at deployment for the organization. This means the environment variable ``MODEL_EXPORT_ENABLED`` should be set to ``True`` in the organization's backend.
```
I'm used to see env variables in all caps, I don't know how much of a standard convention it is.

**Review by SdgJlbl — APPROVED** [review]
Thanks, LGTM

File: docs/source/documentation/concepts.rst (hunk 2 above)
Comment (RomainGoussault) [reply docs/source/documentation/concepts.rst]: Actually it's a helm value and not an environnement variable, my mistake. See here https://artifacthub.io/packages/helm/substra/substra-backend#orchestrator-settings.

**Comment by RomainGoussault** [comment]
@SdgJlbl can you please re-check / approve based on the latest changes, as I have modified the PR since your last approval? Thanks.

**Review by SdgJlbl — APPROVED** [review]

**Pull request merged**